### PR TITLE
WaitSome is deprecated

### PR DESCRIPTION
**Pull request by dieris**
We now only allow sequences with pre-determined length, right? Sorry if I am missing something, my first tests with qgl2

### CODE REVIEW

**Comment by blakejohnson** [comment]
Sorry, if something is in the `attic` directory, then we plan to delete it from the repo. `qgl2_compile_to_hardware` shouldn't need to call `replaceBarriers` at all. So, the correct thing to do here is to simply remove the import.

File: src/python/pyqgl2/main.py [hunk 1]
```diff
@@ -722,7 +722,7 @@ def qgl2_compile_to_hardware(seqs, filename, suffix=''):
     from QGL.PatternUtils import flatten
     from QGL.PulseSequencer import Pulse, CompositePulse
 
-    from pyqgl2.evenblocks import replaceBarriers
+    from attic.evenblocks import replaceBarriers
```
Comment (blakejohnson) [inline src/python/pyqgl2/main.py]: The simple fix here is just to delete this line.

**Comment by blakejohnson** [comment]
It looks like I forgot to push a branch where I had already done this. See `feature/qgl2-compile-to-hardware`.

**Comment by dieris** [comment]
Ooops okay, I'll take that branch then :)

**Pull request closed without merging**